src/main/java/net/glowstone/chunk/GlowChunk.java

```diff
@@ -170,7 +177,17 @@ public GlowBlock getBlock(int x, int y, int z) {
 
     @Override
     public boolean isSlimeChunk() {
-        return false; // TODO: implement slime chunks
+        if (isSlimeChunk == -1) {
+            boolean isSlimeChunk = new Random(this.world.getSeed()
```

Comment:
Can you add a link or source for this formula (e.g. `Taken from the Minecraft Wiki`)